specification/manufacturingdatasolution/mds-service/data-plane/Microsoft.Industries.ManufacturingDataSolution/stable/2025-03-01/openapi.json

```diff
@@ -0,0 +1,3593 @@
+                "format": "uri",
+                "description": "The location for monitoring the operation state."
+              },
+              "background-jobid": {
+                "type": "string",
+                "description": "The background job identifier."
+              }
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/Azure.Core.Foundations.ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "String error code indicating what went wrong."
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DataVersionApis_DeleteVersion": {
+            "$ref": "./examples/DataVersionApis_DeleteVersion.json"
+          }
+        },
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "operation-location"
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/admin/version:switch": {
+      "post": {
+        "operationId": "DataVersionApis_SwitchVersion",
+        "description": "Switch the current data version.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Azure.Core.Foundations.ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The request has been accepted for processing, but processing has not yet completed.",
+            "schema": {
+              "type": "object",
+              "description": "Provides status details for long running operations.",
+              "properties": {
+                "id": {
+                  "type": "string",
+                  "description": "The unique ID of the operation."
+                },
+                "status": {
+                  "$ref": "#/definitions/Azure.Core.Foundations.OperationState",
+                  "description": "The status of the operation"
+                },
+                "error": {
+                  "$ref": "#/definitions/Azure.Core.Foundations.Error",
+                  "description": "Error object that describes the error when status is \"Failed\"."
+                },
+                "result": {
+                  "type": "object",
+                  "description": "The result of the operation.",
+                  "properties": {
+                    "responseHeaders": {
+                      "type": "object",
+                      "properties": {
+                        "parameters": {
+                          "type": "object"
+                        }
+                      },
+                      "required": [
+                        "parameters"
+                      ]
+                    }
+                  },
+                  "required": [
+                    "responseHeaders"
+                  ]
+                }
```

Comment:
This is a very odd result shape. Why return "responseHeaders" here? Should these actually be headers of the response? Also, what are these parameters? How does a user use them?